### PR TITLE
Add a bash helper to generate the version.json file.

### PR DESCRIPTION
**Pull request by Natim**
Following https://github.com/Kinto/kinto/pull/747/files
I made a helper to let us generate the version.json on a tag.

### CODE REVIEW

File: version.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+cd "$(dirname "$0")"
+NAME=$(basename $PWD)
+SOURCE=$(git remote -v | grep origin | grep push | cut -f 2 | sed -e 's|git@|https://|g' | sed -e 's|github.com:|github.com/|g' | sed 's|.git (push)||g')
```
Comment (leplatrem) [inline version.sh]: `$ git config remote.origin.url` ?

**Comment by Natim** [comment]
Updated.

**Comment by leplatrem** [comment]
Since we have a `Makefile` we could use it no? (*Maybe you had something else in mind*)

Also, who and when should we run this ? We could add some instructions in README or elsewhere...

LGTM otherwise! (I was wondering if we should include kinto plugins versions too, using something like `pip freeze | grep kinto`

**Comment by Natim** [comment]
> who and when should we run this?

It would be great to do that on release and bundle it with the wheel.

> I was wondering if we should include kinto plugins versions too, using something like pip freeze | grep kinto)

This was taking care of by other PR on the plugin itselves.
IMHO we are likely to have a wrong information if we use pip freeze here since what was present when the package was tagged might be different from what was installed on the machine.

**Comment by Natim** [comment]
We should probably add the call here: https://github.com/mozilla-services/svcops/blob/167dbd3b53522832ce8880e7d04ff0c9799cac5c/cloudformations/kinto/build_rpm

**Comment by leplatrem** [comment]
I opened https://github.com/mozilla-services/svcops/pull/1249

Closing here

**Pull request closed without merging**